liberty-template/skeleton/catalog-info.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     backstage.io/techdocs-ref: dir:.
     github.com/project-slug: ${{values.repoOwner}}/${{values.repoName}}
     janus-idp.io/tekton : ${{values.component_id}}
-    argocd/app-selector: rht-gitops.com/${{values.component_id}}-dev
+    argocd/app-selector: ${{values.component_id}}-dev
```

Comment:
Should we also add this `openliberty.io/app-kind: OpenLibertyApplication` in the annotations? It's an annotation I made up so that the plugin can (eventually 🤞)determine when to show the `Liberty` tab for a component.